sdk/keyvault/azure-security-keyvault-certificates/src/keyvault_certificates_common_request.cpp

```diff
@@ -17,8 +17,10 @@ _detail::KeyVaultCertificatesCommonRequest::SendRequest(
     Azure::Core::Http::Request& request,
     Azure::Core::Context const& context)
 {
+  request.SetHeader("content-type", "application/json");
```

Comment:
Just in case, check if we have constants for these somewhere. I don't think we do, but just in case.
I do not propose we necessarily create them if we don't.